slides/2024-02-fosdem.md

```diff
@@ -85,19 +82,18 @@ Embedding
 
 ## Future Roadmap
 <div style="width: 50px; height: 10px; background: #488bd1; margin-bottom: 20px;"></div>
-Project maintenance and outreach
-
-CSS Support
 
-Embedding Api
+* Project maintenance and outreach
+* CSS Support
+* Embedding API
+* Initial Android support
 
-Initial Android support
 -----
 <!-- .slide: style="text-align: left;" -->
 
 ## Dioxus (Stylo-Blitz)
 <div style="width: 50px; height: 10px; background: #488bd1; margin-bottom: 20px;"></div>
-Use stylo to Render HTML and CSS
+Use Stylo for CSS styles and selectors matching
```

Comment:
Maybe @mrobinson or @jkelleyrtp can clarify if this wording is accurate.

Reply:
"Use Stylo for CSS styles and selectors matching" sounds good to me!